src/components/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+
+interface IProps {
+  inverted?: boolean;
+  text: string;
+  isColorWhite?: boolean;
+  size?: boolean;
```

Comment:
the name `size` doesn't describe precisely that we're dealing with the boolean.
try naming it something like `isBig` or similar.

Reply:
done